### PR TITLE
edited GDSC BIT

### PR DESCRIPTION
**Pull request by todcru**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import abhinavsri360 from "../../photos/abhinavsri360.jpg";
 import AravindVNair99 from "../../photos/AravindVNair99.jpg";
 import srilaasya from "../../photos/srilaasya.jpg";
 import HarshJain from '../../photos/HarshJain.jpeg'
+import Aswin from "../../photos/Aswin.JPG";
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:6]:
```suggestion
import Aswin from "../../photos/Aswin.jpg";
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@todcru Linux is case-sensitive with files. Hence the build is failing. Your image extension is `jpg`, but your code says `JPG`.

**Comment by todcru** [comment]
> @todcru Linux is case-sensitive with files. Hence the build is failing. Your image extension is `jpg`, but your code says `JPG`.

still its showing some conflict…can you check on it
src/components/json/worldData.js

**Comment by abhinavsri360** [comment]
> > @todcru Linux is case-sensitive with files. Hence the build is failing. Your image extension is `jpg`, but your code says `JPG`.
> 
> still its showing some conflict…can you check on it
> src/components/json/worldData.js

I just fixed it.

**Comment by aravindvnair99** [comment]
> > @todcru Linux is case-sensitive with files. Hence the build is failing. Your image extension is `jpg`, but your code says `JPG`.
> 
> still its showing some conflict…can you check on it
> src/components/json/worldData.js

@todcru That's because you hadn't merged as @abhinavsri360 mentioned here: https://github.com/abhinavsri360/GDSC-Lead-Map/pull/11#issuecomment-817319797

When changes are made upstream, you got to fetch the changes to your fork as well. Anyways, @abhinavsri360 has fixed the merge conflict. You can read more about it here:
- https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/addressing-merge-conflicts
- https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

**Comment by todcru** [comment]
> > > @todcru Linux is case-sensitive with files. Hence the build is failing. Your image extension is `jpg`, but your code says `JPG`.
> > 
> > 
> > still its showing some conflict…can you check on it
> > src/components/json/worldData.js
> 
> @todcru That's because you hadn't merged as @abhinavsri360 mentioned here: [#11 (comment)](https://github.com/abhinavsri360/GDSC-Lead-Map/pull/11#issuecomment-817319797)
> 
> When changes are made upstream, you got to fetch the changes to your fork as well. Anyways, @abhinavsri360 has fixed the merge conflict. You can read more about it here:
> 
> * https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/addressing-merge-conflicts
> * https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

@aravindvnair99 ossm.. thanks mate....yep ill chk into it

**Pull request merged**